packages/render-fidelity-tools/package.json

```diff
@@ -31,9 +31,10 @@
     "@types/resize-observer-browser": "^0.1.2",
     "@types/rimraf": "^2.0.3",
     "@webcomponents/webcomponentsjs": "~2.1.3",
-    "filament": "1.4.5",
+    "filament": "^1.5.0-rc.2",
     "focus-visible": "^5.0.1",
     "fullscreen-polyfill": "^1.0.2",
+    "gl-matrix": "^3.2.1",
```

Comment:
Is this dependency being used anywhere? I wasn't able to find it in the diff..

Reply:
Good catch, this is an indirect dependency so should not be listed here.  Fixed.